src/stores/actions/userAction.js

```diff
@@ -41,10 +47,38 @@ export const getInfoUserAction = () => async (dispatch) => {
   try {
     const data = await getInfoUserService()
     dispatch({
-      type: SET_INFO_USER,
+      type: successAction(GET_USER_INFO),
       payload: data
     })
   } catch (error) {
     throw Error(error)
   }
 }
+
+export const updateInfoUserAction = (payload) => async (dispatch) => {
+  const { data, callback } = payload
+  dispatch({ type: requestAction(UPDATE_USER_INFO) })
+  try {
+    const response = await updateInfoUserService(data)
+    const { hoTen } = response
+    dispatch({
+      type: successAction(GET_USER_INFO),
+      payload: response
+    })
+    dispatch({
+      type: successAction(UPDATE_USER_INFO),
+      payload: { newHoTen: hoTen }
+    })
+    Swal.fire({
+      icon: 'success',
+      title: 'Cập nhật thông tin thành công',
+      text: 'Chúc bạn một ngày tốt lành !',
+      confirmButtonText: 'OK'
+    })
+  } catch (error) {
+    dispatch({ type: failureAction(UPDATE_USER_INFO), error })
+    toast.error(error.response?.data)
```

Comment:
Unnecessary toast.error for handling error. Because it's be handled in axios Client